allennlp/training/learning_rate_schedulers.py

```diff
@@ -135,4 +229,5 @@ def get_lr(self):
         "reduce_on_plateau": torch.optim.lr_scheduler.ReduceLROnPlateau,
         "cosine": torch.optim.lr_scheduler.CosineAnnealingLR,
         "noam": NoamLR,
+        "stlr": STLR
```

Comment:
We already know this is a learning rate, because of the key that's used to get to this name, so "LR" isn't super helpful as a part of the name (see the other names above).  So I'd just use `slanted_triangular` as the name here, and `SlantedTriangular` or `SlantedTriangularLR` as the class name.

Reply:
Done.